src/sentry/static/sentry/app/views/issueList/overview.jsx

```diff
@@ -168,6 +172,14 @@ const IssueList = createReactClass({
       // This can happen when transitionTo is called
       this.fetchData();
     }
+
+    if (
+      prevState.issuesLoading &&
+      !this.state.issuesLoading &&
+      typeof this.props.finishProfile === 'function'
+    ) {
+      this.props.finishProfile();
```

Comment:
You could use `callIfFunction()` to skip the type check :bike:

Reply:
I can follow up with this